.github/workflows/build.yaml

```diff
@@ -43,3 +45,29 @@ jobs:
             type: localBlob
             localReference: extension_definitions.tar.gz
         blobs-directory: blobs.d
+
+  helmchart-imagemap:
+    runs-on: ubuntu-latest
+    needs:
+      - prepare
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - name: Export Helmchart-Imagemap
+        run: |
+          blobs_dir=blobs.d
+          mkdir $blobs_dir
+          tar czf ${blobs_dir}/helmchart_imagemap.tar.gz helmchart-imagemap.yaml
```

Comment:
I think in helmcharts-action (cc-utils), as well as concourse-equivalent, we add the imagemap without wrapping it into a tarchive